releases/release-7.4.0.md

```diff
@@ -0,0 +1,228 @@
+
+    In v7.4.0, TiDB introduce a new system variable [`tidb_opt_objective`](/system-variables.md#tidb_opt_objective-new-in-v740), which controls the estimation method of the optimizer. The default value `moderate` maintains the previous behavior of the optimizer, where it uses runtime statistics to correct estimations based on data modifications. If this system variable is set to `determinate`, the optimizer generates execution plans solely based on statistics without considering runtime corrections.
+
+    For long-term stable OLTP applications or situations where you are confident about existing execution plans, it is recommended to switch to `determinate` mode after testing. This reduces potential plan changes.
+
+    For details, see [user document](/system-variables.md#tidb_opt_objective-new-in-v740)。
+
+
+
+* Enhance the ability to lock statistics [#issue号](链接) @[hi-rustin](https://github.com/hi-rustin) **tw@ran-huang** <!--1557-->
+
+    In v7.4.0, TiDB has enhanced the ability to lock [statistics](/statistics.md#lock-statistics). Now, to ensure operational security, locking and unlocking statistics require the same privileges as granting and collecting statistics. In addition, TiDB also supports locking and unlocking statistics information for specific partitions, improving flexibility. When you have confidence in queries and execution plans in the database and do not want any changes to occur, you can lock statistics to enhance stability.
+
+    For details, see [user document](/statistics.md#锁定统计信息)。
+
+
+### SQL
+
+- note 1
+
+- note 2
+
+### DB operations
+
+- note 1
+
+- note 2
+
+### Observability
+
+- note 1
+
+- note 2
+
+### Data migration
+
+- note 1
+
+* Support real-time update of checkpoint for incremental data validation [#issue号](链接) @[lichunzhu](https://github.com/lichunzhu) **tw@ran-huang** <!--1496-->
+
+    Before v7.4.0, the incremental data validation feature is used to determine whether the data replication from DM to downstream is consistent with upstream. This is used as the basis for cutting over business traffic from upstream database to TiDB. However, due to various factors such as replication delay and waiting for re-validation of inconsistent data, the incremental validation checkpoint must be refreshed every few minutes. This is unacceptable for some business scenarios where the cutover time is only a few tens of seconds.
+
+    With the introduction of real-time updating of checkpoint for incremental data validation, you can pass in the binlog position filled in by the upstream database. Once the incremental validation program detects this binlog position in memory, it will immediately refresh the checkpoint instead of refreshing it every few minutes. Therefore, you can quickly perform cut-off operations based on this immediately returned checkpoint.
```

Comment:
```suggestion
    With the introduction of real-time updating of checkpoint for continuous data validation, you can now provide the binlog position from the upstream database. Once the continuous validation program detects this binlog position in memory, it will immediately refresh the checkpoint instead of refreshing it every few minutes. Therefore, you can quickly perform cut-off operations based on this immediately updated checkpoint.
```